src/js/components/CourseModalComponent.js

```diff
@@ -179,9 +188,9 @@ module.exports = React.createBackboneClass({
       };
     });
 
-    const textbookOptions = this.props.textbooks.map(textbook => {
-      return (<option key={textbook.id} value={textbook.id}>{textbook.get('name')}</option>)
-    });
+    // const textbookOptions = this.props.textbooks.map(textbook => {
```

Comment:
Shouldn't need to comment out lines to remove something as github is a complete record of the code changes.